service/src/main/java/bio/terra/workspace/service/policy/TpsApiDispatch.java

```diff
@@ -87,6 +87,19 @@ public void createPao(UUID workspaceUuid, @Nullable TpsPolicyInputs policyInputs
     }
   }
 
+  // Since policy attributes were added later in development, not all existing
+  // workspaces have an associated policy attribute object. This function creates
+  // an empty one if it does not exist.
+  @Traced
+  public void createPaoIfNotExist(UUID workspaceId) {
```

Comment:
is the idea that we don't backfill but we fill as we encounter them?

if so, should we make `getPaoIfExists` private and always use `createPaoIfNotExist`? is there any case we want to leave the old workspace with empty pao?

Reply:
Yeah, we aren't backfilling but filling in as we encounter them. That's a good idea, let me see what it will take. I don't think there's a case where we'd need to leave the old workspace without a PAO.